<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiPluginManager</class>
 <widget class="QDialog" name="GuiPluginManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="6">
    <widget class="QTreeWidget" name="mp_twPlugins">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLineEdit" name="mp_leFolder"/>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="mp_pbFolder">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="beebeep.qrc">
       <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="mp_pbLoad">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="beebeep.qrc">
       <normaloff>:/images/update.png</normaloff>:/images/update.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="mp_pbEnableAll">
     <property name="text">
      <string>Enable All</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="mp_pbDisableAll">
     <property name="text">
      <string>Disable All</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3" colspan="3">
    <widget class="QPushButton" name="mp_pbClose">
     <property name="text">
      <string>Close</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mp_twPlugins</tabstop>
  <tabstop>mp_leFolder</tabstop>
  <tabstop>mp_pbFolder</tabstop>
  <tabstop>mp_pbLoad</tabstop>
  <tabstop>mp_pbEnableAll</tabstop>
  <tabstop>mp_pbDisableAll</tabstop>
  <tabstop>mp_pbClose</tabstop>
 </tabstops>
 <resources>
  <include location="beebeep.qrc"/>
 </resources>
 <connections/>
</ui>
